ci-support.sh

```diff
@@ -707,10 +709,17 @@ function prepare_downstream_build()
     git commit -a -m "Fake commit to record local changes"
   fi
 
+  url="https://github.com/$user_name/$proj_name.git"
   if [[ "$proj_name" = "mirgecom" ]]; then
-      git clone "https://github.com/illinois-ceesd/$proj_name.git"
+    default_url="https://github.com/illinois-ceesd/$proj_name.git"
   else
-      git clone "https://github.com/inducer/$proj_name.git"
+    default_url="https://github.com/inducer/$proj_name.git"
+  fi
```

Comment:
Hmm. I know what was here before wasn't beautiful, but this logic seems like one step too far. I also know that I'm more likely to say this because it's not me committing the crime.

Realistically, I think the better answer is to permit passing in a `pip`-style git URL: `git+https://...@branchname`. That ought to be easy enough to pick apart?

Reply:
Not easy at all! Bash string slicing and dicing is horrible :fearful: